app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/SponsorPlanEntity.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.droidkaigi.confsched2018.data.db.entity
+
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+
+@Entity(tableName = "sponsor_plan")
+data class SponsorPlanEntity(
```

Comment:
I think we can also delete `SponsorPlanEntity` using `@Embedded`.
What do you think?
https://github.com/DroidKaigi/conference-app-2018/blob/69d8b4282558822281a9a128f20902b88af5219d/app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/SessionEntity.kt#L18
https://github.com/DroidKaigi/conference-app-2018/blob/69d8b4282558822281a9a128f20902b88af5219d/app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/TopicEntity.kt

Reply:
Sorry. I don't know how to delete SponsorPlanEntity using @Embedded.

Reply:
Sorry for lack of explanation. I think that it is possible to delete the DB table and not delete the class. 🙇🏻